### PR TITLE
New oracle helm chart

### PR DESCRIPTION
**Pull request by tkporter**
## Description

This PR moves the old cron-job oracle to be `mock-oracle`. In its old place, a new helm chart for an oracle statefulset has been introduced. All this does is run the oracles in a statefulset on Azure.

Introduced/repurposed some env vars:

```
MOCK_ORACLE_CRON_SCHEDULE - the cron schedule for the old mock oracle
MOCK_ORACLE_DOCKER_IMAGE_REPOSITORY - docker repo for old mock oracle
MOCK_ORACLE_DOCKER_IMAGE_TAG - tag for the image of the old mock oracle

ORACLE_DOCKER_IMAGE_REPOSITORY - docker repo for the new azure based oracle
ORACLE_DOCKER_IMAGE_TAG - tag for the image of the new azure based oracle
ORACLES - number of oracles to deploy in new azure
```

Future work will likely involve pod AAD identities, keyvault integration, and splitting oracle deployments across two kubernetes clusters.

## Other changes

n/a

## Tested

Deployed/destroyed new oracles and `mock-oracles`

## Related issues

n/a

## Backwards compatibility

Instead of using `celotooljs deploy initial|destroy oracle` for the old cronjob oracle, it's now `celotooljs deploy initial|destroy mock-oracle`

## Commit Message
New oracle helm chart

### CODE REVIEW

File: packages/celotool/src/lib/oracle.ts [hunk 1]
```diff
@@ -13,25 +11,18 @@ export async function installHelmChart(celoEnv: string) {
...
-    `--set oracle.image.repository=${fetchEnv(envVar.ORACLE_DOCKER_IMAGE_REPOSITORY)}`,
-    `--set oracle.image.tag=${fetchEnv(envVar.ORACLE_DOCKER_IMAGE_TAG)}`,
-    `--set celocli.nodeUrl=${nodeUrl}`,
-    `--set celocli.image.repository=${fetchEnv(envVar.CELOCLI_STANDALONE_IMAGE_REPOSITORY)}`,
-    `--set celocli.image.tag=${fetchEnv(envVar.CELOCLI_STANDALONE_IMAGE_TAG)}`,
+    `--set environmentName=${celoEnv}`,
+    `--set replicas=4`,
+    `--set image.repository=${fetchEnv(envVar.ORACLE_DOCKER_IMAGE_REPOSITORY)}`,
+    `--set image.tag=${fetchEnv(envVar.ORACLE_DOCKER_IMAGE_TAG)}`,
+    `--set oracle.web3ProviderUrl=http://localhost:8545`,
```
Comment (jcortejoso) [inline packages/celotool/src/lib/oracle.ts:25]: Should we update this to point to the full-node service?

**Review by jcortejoso — APPROVED** [review]
lgtm

File: packages/helm-charts/oracle/templates/statefulset.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  labels:
+{{ include "labels" . | indent 4 }}
+{{ include "oracle-client-component-label" . | indent 4 }}
+spec:
+  clusterIP: None
+  selector:
+{{ include "oracle-client-component-label" . | indent 4 }}
+---
+apiVersion: apps/v1
+kind: StatefulSet
```
Comment (codyborn) [inline packages/helm-charts/oracle/templates/statefulset.yaml:14]: Is this currently being used anywhere or is this for the future?

**Review by codyborn — APPROVED** [review]

File: packages/helm-charts/oracle/templates/statefulset.yaml (hunk 2 above)
Comment (tkporter) [reply packages/helm-charts/oracle/templates/statefulset.yaml:14]: It is currently being used-- it's a statefulset for creating oracle pods 0 through n-1

File: packages/celotool/src/lib/oracle.ts (hunk 1 above)
Comment (tkporter) [reply packages/celotool/src/lib/oracle.ts:25]: I'll add a follow up PR to integrate the two :)

**Pull request merged**